contrib/runners/orquesta_runner/tests/unit/test_with_items.py

```diff
@@ -315,6 +328,11 @@ def test_with_items_concurrency(self):
         lv_ac_db = lv_db_access.LiveAction.get_by_id(str(lv_ac_db.id))
         self.assertEqual(lv_ac_db.status, action_constants.LIVEACTION_STATUS_SUCCEEDED)
 
+    @mock.patch.object(
+        local_shell_command_runner.LocalShellCommandRunner,
+        "run",
+        mock.MagicMock(return_value=RUNNER_RESULT_RUNNING),
+    )
```

Comment:
What happens to this test if we do not patch here?

Reply:
Below test case tries to cancel a task for which the task needs to be in running state. Before this patch, the status was being changed to RUNNING [here](https://github.com/StackStorm/st2/pull/5293/files#diff-4be2f900127f1b9ff8d34fcc0aeb2d585467d3a3c74ba881ebba07291d7c56f7R364). Which will not work with my changes if the task completed before reaching to this line. Therefore, the patch will ensure status being returned RUNNING.